scripts/rv_expression_association/get_rv_outliers.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+
+import click
+import logging
+import hail as hl
+import pandas as pd
+from cpg_utils.hail_batch import dataset_path, init_batch, output_path, reference_path
+from cloudpathlib import AnyPath
+
+# use logging to print statements, display at info level
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+MT = dataset_path('mt/v7.mt')
+
+
+# get OneK1K sample ID and gene name as arguments using click
+@click.command()
+@click.option('--onek1k-id', required=True)
+@click.option('--gene-name', required=True)
+@click.option('--sample-mapping-file', required=True)
+@click.option('--gene-file', required=True)
+@click.option(
+    '--window-size',
+    required=True,
+    help='the size of the flanking region, to be added twice, on either side of the gene body',
+)
+def main(
+    onek1k_id: str,
+    gene_name: str,
+    sample_mapping_file: str,  # 'scrna-seq/grch38_association_files/OneK1K_CPG_IDs.tsv'
+    gene_file: str,  # e.g., 'scrna-seq/grch38_association_files/gene_location_files/GRCh38_geneloc_chr1.tsv'
```

Comment:
How about using https://hail.is/docs/0.2/experimental/index.html#hail.experimental.import_gtf instead?

Reply:
https://hail.is/docs/0.2/experimental/index.html#hail.experimental.get_gene_intervals might be even easier.

Reply:
that would be just to find the start / end of the gene, I'd still need to define the interval?

Reply:
ah you're saying as an alternative to the gene_file! That would work actually, I'd also not need to specify the right chromosome every time! Do we have access to a Gencode GTF?

Reply:
Yes, we do, however there's some related discussion on how best to read it: https://centrepopgen.slack.com/archives/C018KFBCR1C/p1661409100527349?thread_ts=1658208231.938539&cid=C018KFBCR1C